crates/common/src/file.rs

```diff
@@ -0,0 +1,28 @@
+use std::{fs::File, path::Path};
+
+use reqwest::blocking as reqwest;
+use zip::CompressionMethod;
+use zip_extensions::ZipWriterExtensions;
+
+pub fn download_file<S: Into<String>, P: AsRef<Path>>(source: S, dest: P) -> anyhow::Result<()> {
+    let mut response = reqwest::get(source.into())?;
+    let mut file = File::create(dest.as_ref())?;
+    let _ = &response.copy_to(&mut file)?;
+    Ok(())
+}
+
+#[allow(deprecated)]
```

Comment:
what's the deprecation?

Reply:
Originally I was using the `zip` crate directly, rather than zip_extensions. Under `zip` all the `..._from_path(...)` functions are deprecated because they strip '..' from the path instead of handling it. I have opinions, but they are not relevant here 😆

Now it is a leftover I forgot to remove. That will be corrected in the next push.